parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -915,14 +984,12 @@ pub mod pallet {
 		}
 
 		fn store_execution_header(
-			block_hash: H256,
 			header: ExecutionHeader,
 			beacon_slot: u64,
 			beacon_block_root: H256,
-		) {
+		) -> DispatchResult {
```

Comment:
Why does this method need to return a dispatch result? I don't see an error condition that can arise in the method, unless I am missing it?